source/documentation/managing_apps/autoscaling.erb

```diff
@@ -0,0 +1,282 @@
+## Autoscaling
+
+Apps can be configured to dynamically scale in and out based on autoscaling
+policies.
+
+Scaling your app to fewer instances can reduce the cost during periods of low
+usage,
+eg at night.
+
+Scaling your app based on throughput or average latency can improve the user
+experience of your app during periods of high traffic,
+eg before a submission deadline.
+
+### Scaling methods
+
+There are two ways of scaling:
+
+1. `schedules` - scale apps based on time of day and day of week or month
+1. `scaling_rules` - scale apps based on metrics, eg average latency
+
+Schedules and scaling rules can be used at the same time, or on their own.
+
+### Limits
+
+There are two limits which you can apply to an autoscaling policy:
+
+* `instance_max_count` - maximum number of instances to run
+* `instance_min_count` - minimum number of instances to run
+
+The maximum value for `instance_max_count` is determined by your [quota](#quotas).
+
+The minimum value for `instance_min_count` is 1.
+
+### Metrics
+
+You can scale your app based on some provided metrics:
+
+* `memoryused` - how many megabytes of memory an app instance is using
+* `memoryutil` - percentage of memory utilization where `50` is 50%
+* `cpu` - percentage of CPU utilization where `100` is 100% of a single CPU
+* `responsetime` - average time in milliseconds an app instance takes to respond to external HTTP requests
+* `throughput` - number of external HTTP requests processed per second
+
+You can also scale your app based on custom metrics, please refer to the
+[app-autoscaler user guide](https://github.com/cloudfoundry/app-autoscaler/blob/develop/docs/Readme.md#auto-scale-your-application-with-custom-metrics)
```

Comment:
Is this document safe to link to? It's on the `develop` branch. Should we link to the tag we deployed?

Reply:
Resolved in 6877c6a